workers.go

```diff
@@ -71,6 +72,15 @@ func (n *Node) processModuleEvents(
 	// This is only for debugging / diagnostic purposes.
 	n.interceptEvents(plainEvents)
 
+	// In Trace mode, log all events.
+	if n.Config.Logger.Level() <= logging.LevelTrace {
```

Comment:
See other comment.

In this particular case, an alternative would be to add tracing to the interceptor or use a "`ConsoleInterceptor`" (in case there is no interceptor) if we want to trace the events. The choice of interceptor could be driven by the same command-line argument that already drives the "verbousness" of the logger.